tests/ApplicationInsightsCompatibility/Intercept.Latest.Tests/Intercept.Latest.Tests.csproj

```diff
@@ -15,11 +15,12 @@
     <AssemblyName>Intercept.Latest.Tests_x64</AssemblyName>
   </PropertyGroup>
   <PropertyGroup>
-    <Platforms>x64;x86;AnyCPU</Platforms>
+    <Platforms>x64;x86;AnyCPU;ARM64</Platforms>
     <RootNamespace>Intercept.Latest.Tests</RootNamespace>
     <AssemblyName>Intercept.Latest.Tests_$(Platform)</AssemblyName>
     <TargetFramework>net45</TargetFramework>
     <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <PlatformTarget Condition="'$(Platform)'=='ARM64'">arm64</PlatformTarget>
```

Comment:
nit: use the same casing for "arm64" in PlatformTarget properties (either all upper case or all lower case)

Reply:
oops, fixing